_data/versions.yml

```diff
@@ -289,6 +289,46 @@ v2.2:
     networking-calico:
       version: 1.4.2
       url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=1.4.2
+v2.3:
```

Comment:
I'm not sure what has to be done to keep it from being there.  Check the RELEASING.md, it goes through that.